packages/deno/src/generators/application/application.ts

```diff
@@ -0,0 +1,34 @@
+import { formatFiles, installPackagesTask, Tree } from '@nrwl/devkit';
+import { initDeno } from '../init/generator';
+import denoSetupServerless from '../setup-serverless/setup-serverless';
+import { addPathToDenoSettings } from '../utils/add-path';
+import { addFiles, addProjectConfig, normalizeOptions } from './lib';
+import { DenoAppGeneratorSchema } from './schema';
+
+export async function denoApplicationGenerator(
+  tree: Tree,
+  options: DenoAppGeneratorSchema
+) {
+  const normalizedOptions = normalizeOptions(tree, options);
+
+  await initDeno(tree);
+  addProjectConfig(tree, normalizedOptions);
+  addFiles(tree, normalizedOptions);
+  addPathToDenoSettings(tree, normalizedOptions.projectRoot);
+
+  if (options.platform && options.platform !== 'none') {
+    await denoSetupServerless(tree, {
+      platform: options.platform,
+      project: normalizedOptions.projectName,
+    });
+    // TODO(caleb): handle making netlify config for making a deno app on netlify
```

Comment:
We should handle this in this PR I think?

Reply:
Potentially, this was a question I had last minute on Friday. We could technically do it in the --framework flag PR that'll be after this if we didn't want add to this PR since it's kind of a extra add on I didn't anticipate. I'm fine adding it to this PR but wanted some feedback to see if this was even the correct path to take